application.go

```diff
@@ -99,6 +98,8 @@ type Application struct {
 	LastTaskFailure       *LastTaskFailure        `json:"lastTaskFailure,omitempty"`
 	Fetch                 *[]Fetch                `json:"fetch,omitempty"`
 	IPAddressPerTask      *IPAddressPerTask       `json:"ipAddress,omitempty"`
+	Env                   map[string]string       `json:"env,omitempty"`
```

Comment:
Can we please return the `Env` definition to the location where it was before (i.e., line 67) in order to minimize the diff?